src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -280,6 +283,24 @@ public bool HasPrivateKey
             }
         }
 
+        /// <summary>
+        /// Gets a bool that determines if the 'key_ops' (Key Operations) property should be serialized.
+        /// </summary>
+        /// <return>true if 'key_ops' (Key Operations) is not empty; otherwise, false.</return>
+        public bool ShouldSerializeKeyOps()
```

Comment:
Where is this used?

Reply:
I believe this is called automatically when serialization occurs. I just followed the instructions here:
https://www.newtonsoft.com/json/help/html/ConditionalProperties.htm

Reply:
Can you add a to the comments that JSON.Net uses this.

Reply:
Just added this in for both methods.